sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -74,6 +77,9 @@
             IntegrationTestRepoConfig(online_store=REDIS_CONFIG),
             IntegrationTestRepoConfig(online_store=REDIS_CLUSTER_CONFIG),
             # GCP configurations
+            IntegrationTestRepoConfig(
```

Comment:
probably would not include them here. we can make a separate make command that tests third party stores too though.